ctapipe_io_lst/__init__.py

```diff
@@ -184,51 +197,106 @@ def fill_lst_event_container_from_zfile(self, event):
         event_container.ped_id = event.ped_id
         event_container.module_status = event.lstcam.module_status
         event_container.extdevices_presence = event.lstcam.extdevices_presence
-        event_container.tib_data = event.lstcam.tib_data
-        event_container.cdts_data = event.lstcam.cdts_data
+
+        # unpack TIB data
+        rec_fmt = '=IHIBB'
+        rec_unpack = struct.Struct(rec_fmt).unpack_from
+        unpacked_tib = rec_unpack(event.lstcam.tib_data)
+        event_container.tib_event_counter = unpacked_tib[0]
+        event_container.tib_pps_counter = unpacked_tib[1]
+        event_container.tib_tenMHz_counter = unpacked_tib[2]
+        event_container.tib_stereo_pattern = unpacked_tib[3]
+        event_container.tib_masked_trigger = unpacked_tib[4]
+        #event_container.cdts_data = event.lstcam.cdts_data
         event_container.swat_data = event.lstcam.swat_data
-        event_container.counters = event.lstcam.counters
+
+        # unpack CDTS data
+        rec_fmt = '=IIIQQBBB'
+        rec_unpack = struct.Struct(rec_fmt).unpack_from
```

Comment:
why not simpy `struct.unpack(rec_fmt, event.lstcam.cdts_data)`

Reply:
No specific reason, if it makes the same I can change it